src/test/java/teammates/test/cases/ui/StudentFeedbackSubmissionEditSaveActionTest.java

```diff
@@ -817,7 +844,8 @@ public void testExecuteAndPostProcess() throws Exception {
                 Const.ParamsNames.FEEDBACK_RESPONSE_TEXT + "-1-0", "100",
                 Const.ParamsNames.FEEDBACK_QUESTION_NUMSCALE_MIN + "-1-0", Integer.toString(fqd.getMinScale()),
                 Const.ParamsNames.FEEDBACK_QUESTION_NUMSCALE_MAX + "-1-0", Integer.toString(fqd.getMaxScale()),
-                Const.ParamsNames.FEEDBACK_QUESTION_NUMSCALE_STEP + "-1-0", StringHelper.toDecimalFormatString(fqd.getStep())
+                Const.ParamsNames.FEEDBACK_QUESTION_NUMSCALE_STEP + "-1-0",
+                        StringHelper.toDecimalFormatString(fqd.getStep())
```

Comment:
I don't recall this line being stopped by Checkstyle. If you want to fix this, don't do it here.